backend/src/types/payments.ts

```
@@ -333,6 +339,52 @@ export interface RazorpayOrderRow {
+  raw: Record<string, unknown>;
+  createdAt: Date | string;
+  updatedAt: Date | string;
+}
+
+export type PaystackTransactionStatus =
+  | 'initialized'
+  | 'pending'
+  | 'success'
+  | 'failed'
+  | 'abandoned'
+  | 'reversed';
```

ESLint

[error] Replace `|·'initialized'⏎··|·'pending'⏎··|·'success'⏎··|·'failed'⏎··|·'abandoned'⏎·` with `'initialized'·|·'pending'·|·'success'·|·'failed'·|·'abandoned'`

(prettier/prettier)